### PR TITLE
Creating administrator account

### PR DESCRIPTION
**Pull request by roller1187**

### CODE REVIEW

File: roles/gitea-ocp/tasks/main.yml [hunk 1]
```diff
@@ -78,3 +78,21 @@
...
   delay: 10
   ignore_errors: yes
+
+- name: Get Gitea pod info
+  k8s_info:
+    api_version: v1
+    kind: Pod
+    namespace: '{{ _gitea_namespace }}'
+    label_selectors:
+      - "app=gitea"
```
Comment (andykrohg) [inline roles/gitea-ocp/tasks/main.yml]: This should be `{{ _gitea_name }}`

File: roles/gitea-ocp/defaults/main.yml [hunk 2]
```diff
@@ -3,6 +3,10 @@ _gitea_state: present
 _gitea_namespace: gitea
 _gitea_name: gitea
 
+_gitea_admin_user: administrator
+_gitea_admin_password: openshift
```
Comment (andykrohg) [inline roles/gitea-ocp/defaults/main.yml]: Can we place this in an openshift secret as in `nexus-operator`?

**Review by andykrohg — APPROVED** [review]
lgtm!

**Comment by andykrohg** [comment]
Actually, this needs a version bump in `operate.conf`

**Review by jflowers — APPROVED** [review]
lgtm

File: roles/gitea-ocp/defaults/main.yml (hunk 2 above)
Comment (roller1187) [reply roles/gitea-ocp/defaults/main.yml]: yes we can!

**Comment by andykrohg** [comment]
lgtm! good to merge when test passes 👍

**Pull request merged**